src/app/app.module.ts

```diff
@@ -13,6 +13,7 @@ import { UserService } from './shared/user.service';
 import { ValidatorService } from './validators/validator.service';
 
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { PlanetFormsModule } from './shared/planet-forms.module';

```

Comment:
do we need this planet-form in app module?

Reply:
No, we can remove this now.